docs/pages/includes/install-linux.mdx

```diff
@@ -120,15 +120,14 @@ Next, use the appropriate commands for your environment to install your package.
 For example, use the following commands to install Teleport on a machine with an
 x86-64 chip via tarball:
 
-
 ```code
 $ curl https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256
 # <checksum> <filename>
 $ curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
 $ shasum -a 256 teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
 # Verify that the checksums match
-$ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
```

Comment:
What's wrong with `xzf`? It's not strictly required, as modern tar implementations usually detect compression automatically, but it's not incorrect either.

>      -z, --gunzip, --gzip
>             (c mode only) Compress the resulting archive with gzip(1).  In extract or list modes, this option is ignored.  Note that this tar
>             implementation recognizes gzip compression automatically when reading archives.